src/styles/_color-themes.scss

```diff
@@ -78,6 +78,7 @@ $themes: (
     primary--1-bg--7: var(--sendbird-light-primary-100),
     secondary--3-2: var(--sendbird-light-secondary-300),
     error--3-2: var(--sendbird-light-error-300),
+    error--4-1: var(--sendbird-dark-error-400),
```

Comment:
it seems that dark palette colors have been used in the light theme. Is that intentional?

Reply:
oh it should be light although they have same color! thanks for the catch